src/Router.js

```diff
@@ -1,10 +1,11 @@
-import React from "react";
+import React from 'react'
```

Comment:
This made a lot of unnecessary changes, you must have set it to use "(double quotes) and not ' (single quotes), you made a lot of useless changes.

Reply:
Well, that was my extension in Prettier at the time I applied it, but the correct thing would be to have defined this in the root of the project in a configuration file to standardize the code formatting